src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -993,6 +993,9 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 
 		this._chatEditsActionsDisposables.add(this.instantiationService.createInstance(MenuWorkbenchButtonBar, actionsContainer, MenuId.ChatEditingWidgetToolbar, {
 			telemetrySource: this.options.menus.telemetrySource,
+			menuOptions: {
```

Comment:
Is this guaranteed to rerender when the session id changes?

Reply:
Yeah, if anything it's rerendering a bit too much right now, gotta fix that in main too